app/assets/javascripts/admin/payments/services/payment.js.coffee

```diff
@@ -30,11 +30,22 @@ angular.module('admin.payments').factory 'Payment', (AdminStripeElements, curren
               month: @form_data.card.exp_month
               year: @form_data.card.exp_year
           }
+        when 'stripe_sca'
```

Comment:
Is duplicating this block necessary? It's identical to the one above.

Reply:
I dont know the exact syntax for the when with two cases...

but actually, this answers to the other comments below: these changes are optimized for deletion of the stripe connect method very soon.

do you mind if I leave my version, basically separate lines for each method? it will be straight forward to delete them soon.